### ISSUE TITLE
Indicate Streamlink errors on exit

### ISSUE THREAD

**Issue by leon-richardt** [Chatterino/chatterino2#2847]
**Describe your issue**
When attempting to open a channel in Streamlink, no errors are reported back to the user when launching Streamlink fails.

**OS and Chatterino Version**
Arch Linux (5.12.8-arch1-1), Chatterino 2.3.2 (commit 57354283)

**Comment by leon-richardt** [comment]
"Classic" solutions for these kind of things:
1. Show a system message with the Streamlink error in chat
2. Show a poup with the Streamlink

Personally, I'd prefer solution 1.

**Comment by Mm2PL** [comment]
I'd generally prefer solution 1, however this won't work in faster chats.

**Comment by leon-richardt** [comment]
Hint for possible implementation: We could pass `--loglevel error`  to Streamlink in order to restrict output to error messages.

**Comment by apa420** [comment]
I'd prefer solution 2 a lot more.
If you are having issues with streamlink I'd like the entire log of what's wrong instead of just "it doesn't work" or something similar so I can debug it.

**Comment by leon-richardt** [comment]
> the entire log

How long are Streamlink error messages usually? Every message I have encountered so far would likely fit in one or two chat lines

**Comment by apa420** [comment]
You are correct, from the error message I've seen they're usually only one line long so it could fit in twitch chat.

The most common error a person would most likely get is the `stream is not available at` error because either twitch or the streamer is having issues.
Would this fit better into a chat or as an error message?

To me debugging something through the help of twitch chat is a strange idea and a bit unintuitive.

**Comment by leon-richardt** [comment]
I might have misunderstood you. I read your original comment as "A popup is preferable because a lengthy error log would be more readable in a popup than a chat message". I would generally agree with this but in the case of Streamlink, error messages really aren't that lengthy.

Nonetheless, I would now prefer a popup simply because it will not be missed in fast chats.
